src/documents/ModelTransformationAPI.md

```diff
@@ -3217,3 +3224,386 @@ transform: {
+        "foo": "bar"
+    }
+}
+</code></pre></td>
+</tr>
+</tbody>
+</table>
+
+### fluid.transforms.objectToJSONString
+
+**Type:** standardTransformFunction
+
+**Description:** Transform an object to a string using `JSON.stringify`.
+
+**Invertibility:** Partly invertible via `fluid.transforms.JSONstringToObject`.
+
+**Syntax:**
+```
+{
+    "transform": {
+        "type": "fluid.transforms.objectToJSONString",
+        "inputPath": "",
+        "outputPath": "string"
+    }
+}
+```
+
+#### Examples:
+
+**Example 1: Convert an object payload into a string payload.**
+
+<table><thead>
+</thead><tbody>
+<tr><th>source</th><th>rule</th><th>Output</th></tr>
+<tr><td><pre><code>
+{
+    "object": {
+        "foo": "bar"
+    }
+}
+</code></pre></td>
+<td><pre><code>
+{
+    "transform": {
+        "type": "fluid.transforms.objectToJSONString",
+        "inputPath": "object",
+        "outputPath": "string"
+    }
+}
+</code></pre></td>
+<td><pre><code>
+{
+    "string": "{\"foo\":\"bar\"}"
+}
+</code></pre></td>
+</tr>
+</tbody>
+</table>
+
+**Example 2: Add spaces and carriage returns to the stringified output.**
+
+You can pass the `space` option to control whether spaces are included between keys and values in the string output.  A
+positive value also results in carriage returns between key/value pairs.  The default value for `space` is 0, which
+disables spacing and line breaks.
+
+<table><thead>
+</thead><tbody>
+<tr><th>source</th><th>rule</th><th>Output</th></tr>
+<tr><td><pre><code>
+{
+    "object": {
+        "foo": "bar"
+    }
+}
+</code></pre></td>
+<td><pre><code>
+{
+    "transform": {
+        "type": "fluid.transforms.objectToJSONString",
+        "inputPath": "object",
+        "outputPath": "string",
+        "space": 2
+    }
+}
+</code></pre></td>
+<td><pre><code>
+{
+    "string": "{\n  \"foo\":  \"bar\"\n}"
+}
+</code></pre></td>
+</tr>
+</tbody>
+</table>
+
+### fluid.transforms.stringToDate
+
+**Type:** standardTransformFunction
+
+**Description:** Transform a string to a date using the Date constructor.  Accepts (among other things) the date and dateTime values returned by HTML5 date and dateTime inputs. Note: This function allows you to create Date objects from an ISO 8601 string such as `2017-01-23T08:51:25.891Z`.  It is intended to provide a consistent mechanism for recreating Date objects stored as strings.  Although the framework currently works as expected with Date objects stored in the model, this is very likely to change.  If you are working with Date objects in your model, your best option for ensuring your code continues to work in the future is to handle serialisation and deserialisation yourself, for example, by using this transform and one of its inverse transforms, `fluid.transforms.dateToString` or `fluid.transforms.dateTimeToString`.  See [the Infusion documentation](http://docs.fluidproject.org/infusion/development/FrameworkConcepts.html#model-objects) for details about supported model values.
```

Comment:
I don't think that what you mean by "handle serialisation and deserialisation yourself" is very clear. The principal risk that people face at the moment is that i) users of Date objects often tend to treat them as mutable (given they have a mutable API) whereas ii) the framework simply ships them around as if they were immutable - that is, the same Date object will end up embedded in multiple model trees. Assuming that the user does not try to modify the particular Date objects that they find or put within models, everything will be fine. Given this, "works as expected" is a bit of a hedge. I'd be happy to just take on this wording and edit it myself if you're ok with that, since I know we have had a few passes at it already.